src/Proxy/FilterPathProxy.php

```diff
@@ -17,7 +16,7 @@
 use ValueObjects\Web\Domain;
 use ValueObjects\Web\PortNumber;
 
-class CalendarSummaryProxy extends Proxy
+class FilterPathProxy extends Proxy
```

Comment:
Eventueel aanpassen naar iets in de trend van `FilterGetPathProxy`, of de method configureerbaar maken? Is momenteel hardcoded op `GET` in `createFilter()`.